doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -278,10 +278,13 @@ There is dedicated syntax for list and array literals.
    ltac2_expr0 ::= ( @ltac2_expr )
    | ( @ltac2_expr : @ltac2_type )
    | ()
-   | [ {*; @ltac2_expr5 } ]
+   | @array_literal
+   | @list_literal
    | %{ @ltac2_expr0 with {? {+; @tac2rec_fieldexpr } {? ; } } %}
    | %{ {? {+; @tac2rec_fieldexpr } {? ; } } %}
    | @ltac2_tactic_atom
+   array_literal ::= [ %| {*; @ltac2_expr5 } %| ]
+   list_literal ::= [ {*; @ltac2_expr5 } ]
```

Comment:
It would be more readable to inline these rather than have the new nonterminals.  If you don't want to change the mlg file, add something like the following to `common.edit_mlg` to change it only in the doc:

```
| array_literal
| list_literal
] (* end SPLICE *)
```

(third line already in the file)

Reply:
Ah that's neat, thanks for the pointer :+1:  I just did that in my latest commit!